source/features/reactions-avatars.tsx

```diff
@@ -21,7 +21,7 @@ interface Participant {
 
 function getParticipants(button: HTMLButtonElement): Participant[] {
 	const currentUser = getUsername();
-	const users = button.getAttribute('aria-label')!
+	const users = (pageDetect.isReleasesOrTags() ? button.getAttribute('title') : button.getAttribute('aria-label'))!
```

Comment:
The list of people who reacted is now in the `title` attribute (which makes sense 'cause you can hover on the button to see it).
…But only for releases pages, apparently.

Reply:
Does GHE has reactions? Maybe it would be better to `button.getAttribute('title') ?? button.getAttribute('aria-label')`

Reply:
> Maybe it would be better to `button.getAttribute('title') ?? button.getAttribute('aria-label')`

Can't, because the buttons on PR/issue comments also have a title attribute (but it's empty). And the buttons on releases also have an `aria-label` — but it doesn't contain the right data for the feature.

Reply:
Aren't the code correct in those conditions?

- PR/issue comments: `title` is empty so it uses `aria-label` (correct)
- Releases: `title` is preffered

Reply:
> `title` is empty so it uses `aria-label`

No, `??` (or its compiled equivalent) triggers only if the left value is `null` or `undefined`, it won't coerce an empty string into a falsy value like `||` would. We could use that last one, but I think the tertiary is clearer about why having both attributes is needed.

Reply:
Right, I missed the difference between `||` and `??` (which I don't really use outside of RGH due to the fact it's still a shiny new thing) 😅

Reply:
> Also let's wait and see if GitHub changes this somewhere else. It's also strange they only do this on one page.

Seems like you were right, it changed for review comments too! But not regular comments… :grimacing: 

> which I don't really use outside of RGH due to the fact it's still a shiny new thing

Not so much, it's supported by all major browsers now, and by Node since v14.